src/document.rs

```diff
@@ -18,8 +18,30 @@ use crate::cobalt_model;
 use crate::cobalt_model::files;
 use crate::cobalt_model::permalink;
 use crate::cobalt_model::slug;
+use crate::cobalt_model::Minify;
 use crate::error::*;
 
+pub struct RenderContex<'a> {
+    pub parser: &'a cobalt_model::Liquid,
+    pub markdown: &'a cobalt_model::Markdown,
+    pub globals: &'a Object,
+    pub minify: Minify,
+}
+
+#[cfg(not(feature = "html-minifier"))]
+fn minify_if_enabled(html: String, _context: &RenderContex) -> Result<String> {
+    Ok(html)
+}
+
+#[cfg(feature = "html-minifier")]
+fn minify_if_enabled(html: String, context: &RenderContex) -> Result<String> {
+    if context.minify.html {
```

Comment:
We might render other types of data than html

Ideas:
- Limit this it `.html` extension
- Switch control for this over to the frontmatter.  We'd then move the other minification options to be under `assets` in the config.

Thoughts?

Reply:
I see, yes i will try to add a double check that the final asset is actually an html file.